extensions/src/ACRE2Core/Wave.cpp

```diff
@@ -33,14 +33,12 @@ CWave::~CWave(void)
 
 BOOL CWave::Load(std::string wavFile)
 {
-    BOOL bResult = FALSE;
-
-
+    BOOL bResult = false;
 
     std::ifstream file;
     file.open(wavFile, std::fstream::in | std::fstream::binary);
     if (file.bad() || !file.is_open())
-        return FALSE;
+        return false;
```

Comment:
If a function returns BOOL, I would expect it to return TRUE/FALSE and not true/false.
```suggestion
        return FALSE;
```